src/test/java/Leas_Liudmila/junit/AdoptionInTest.java

```diff
@@ -0,0 +1,65 @@
+package Leas_Liudmila.junit;
+
+import Leas_Liudmila.LLAdoptPage;
+import Leas_Liudmila.junit.actions.AdoptPageActions;
+import helpers.Helpers;
+import org.apache.commons.lang.RandomStringUtils;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import java.util.ArrayList;
+import java.util.concurrent.TimeUnit;
+
+import static helpers.Helpers.stepResults;
+
+public class AdoptionInTest {
+    WebDriver driver = null;
+    AdoptPageActions myActions = null;
+    WebDriverWait wait = null;
+    LLAdoptPage myPageLL = null;
+
+    @BeforeEach
```

Comment:
Please create a BaseTest class and put there BeforeEach hook method, otherwise it is copypasted everywhere

Reply:
and extend your test classes from BaseTest